tests/api.c

```diff
@@ -29200,8 +29200,15 @@ static int test_wolfSSL_GetLoggingCb (void)
             wolfSSL_SetLoggingCb(NULL);
         }
     }
-    printf(resultFmt, ret == 0 ? passed : failed);
 #endif
+    if (ret == 0) {
+        if (wolfSSL_GetLoggingCb() == NULL) {
+            ret = 0;
```

Comment:
How do we detect an error in this case, suppose if wolfSSL_GetLoggingCb() did not return NULL?  Looks like we need an else block, or to use the inverse.